src/python_testing/TestConformanceSupport.py

```diff
@@ -15,6 +15,9 @@
 #    limitations under the License.
 #
 
+# test-runner-runs: run1
+# test-runner-run/run1/script-args: --trace-to json:${TRACE_TEST_JSON}.json --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
```

Comment:
This one is odd: why do we even run it as a test runner if it has no "app" to test at all? it seems like a self-test.

I believe we should separate these out somehow. Please add an issue for this and we can talk to @cecille about this.

Reply:
Fixed.